pkgs/development/python-modules/textual-dev/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, aiohttp
+, buildPythonPackage
+, click
+, fetchFromGitHub
+, msgpack
+, poetry-core
+, pythonOlder
+, textual
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "textual-dev";
+  version = "1.2.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "Textualize";
+    repo = pname;
```

The repo name and pname are not considered the same on nixpkgs (:

```suggestion
    repo = "textual-dev";
```

---

I am not sure I follow this. I was basing my derivation on this one [here](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/development/python-modules/textual/default.nix#L80) of textual. In that derivation they use `pname` for the `repo` name?

---

A lot of packages write it with pname, but it is actually not considered the same and hence my suggestion. (Based on multiple reviews that I've received)

---

Ah okay thanks again, I've set `repo = "textual-dev"` explicitly